sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -254,12 +256,25 @@ def test_int_overflow_mutual_info_fowlkes_mallows_score():
 
 
 def test_entropy():
-    ent = entropy([0, 0, 42.0])
-    assert_almost_equal(ent, 0.6365141, 5)
+    assert_almost_equal(entropy([0, 0, 42.0]), 0.6365141, 5)
```

Comment:
I find this test case weird on implicitly typed floating point values, because the docstring for the entropy method states that it expects integers.

But since it works and it is tested, let's keep it for now and let's try to make the array API variants behave similarly to the existing NumPy code.